Makefile

```diff
@@ -71,14 +43,6 @@ commit-hook:
 full-clean: check-docker
 	docker images -q $(IMAGE_PREFIX)$(COMPONENT) | xargs docker rmi -f
 
-postgres:
```

Comment:
Can we leave this target in? I use `make postgres` all the time, so I can run `make test-unit`...